README.md

```diff
@@ -3,3 +3,13 @@
 
 # telemetry-streaming
 Spark Streaming ETL jobs for Mozilla Telemetry
+
+## Development
+
+The recommended workflow for running tests is to use your favorite editor for editing
+the source code and running the tests via sbt. Some common invocations for sbt:
+
+* `sbt test  # run the basic set of tests (good enough for most purposes)`
+* `sbt "testOnly *ErrorAgg*"  # run the tests only for packages matching ErrorAgg`
+* `sbt "testOnly *ErrorAgg* -- -z version"  # run the tests only for packages matching ErrorAgg, limited to test cases with "version" in them`
```

Comment:
Add also `sbt dockerComposeTest`, as well as `sbt "dockerComposeTest -tags:DockerComposeTag"` which will only run tests tagged with `DockerComposeTag`. This will matter more when I merge #80

Reply:
Ok, I'll add those!